input-output-hk/cardano-sl
# [DEVOPS-398] Update testnet launch genesis data

## Description

Fix the testnet genesis parameters total balance and block version data.

## Linked issue

https://iohk.myjetbrains.com/youtrack/issue/DEVOPS-398

## Type of change


## QA Steps

    nix-build -A demoClusterLaunchGenesis -o demo-cluster-launch-genesis.sh
    ./demo-cluster-launch-genesis.sh
    (check logs to see that it works ok)

Current testnet is running with this genesis data. Poor addresses have 19,999,999.999999 Ada and fake AVVM seeds redeem 20,000,000 Ada.
